UnityProject/Assets/Scripts/Systems/Occupations/Occupation.cs

```diff
@@ -149,6 +151,10 @@ public enum OccupationDifficulty
 	[SerializeField] private SerializableDictionary<string, bool> customProperties = default;
 	public SerializableDictionary<string, bool> CustomProperties => customProperties;
 
+	[Header("Custom properties that will be applied\nto new bodies with this occupation")]
+	[SerializeReference, SelectImplementation(typeof(OccupationCustomEffectBase))] public List<OccupationCustomEffectBase> BetterCustomProperties = new List<OccupationCustomEffectBase>();
```

Comment:
I wish this was called something different that made more sense after the lesserCustomProperties is eventually gone.

Reply:
it's not too bad

Reply:
I digress